controllers/namespace_controller.go

```diff
@@ -154,6 +150,44 @@ func (r *NamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		},
 	}
 
+	allowedAPIGroups, err := getDFDSCrossplaneAPIGroups("DFDS_CROSSPLANE_PKG_ALLOWED_API_GROUPS", []string{"xplane.dfds.cloud"})
```

Comment:
we might need to include all subdomains for the API groups as well but we will need to test it in the sandbox cluster first

Reply:
This is just a fallback so if no value is set, one gets used. Needed values are passed in by the Helm chart so they are not hardcoded